docs/dashboard.asciidoc

```diff
@@ -3,66 +3,86 @@
 
 [partintro]
 --
-A Kibana _dashboard_ displays a collection of visualizations, searches, and {kibana-ref}/maps.html[maps].
-You can arrange, resize, and edit the dashboard content and then save the dashboard
-so you can share it.
 
-[role="screenshot"]
-image:images/Dashboard_example.png[Example dashboard]
+A {kib} _dashboard_ is a collection of visualizations, searches, and 
+maps, typically in real-time. Dashboards provide 
+at-a-glance insights into your data and enable you to drill down into details. 
 
---
+To start working with dashboards, click *Dashboard* in the side navigation. 
+You can:
 
-[[dashboard-getting-started]]
-== Building a Dashboard
+* <<dashboard-create-new-dashboard, Create a dashboard>>
+* <<customizing-your-dashboard, Arrange dashboard elements>>
+* <<sharing-dashboards, Share a dashboard>>
+* <<import-dashboards, Import and export dashboards>>
+* <<viewing-detailed-information, Inspect and edit dashboard elements>>
+
+
+[role="screenshot"]
+image:images/Dashboard_example.png[Example dashboard]
 
-If you haven't yet indexed data into {es} or created an index pattern,
-you'll be prompted to do so as you follow the steps for creating a dashboard.
-Or, you can use one of the prebuilt sample data sets, available from the
-Kibana home page.
 
 [float]
 [[dashboard-read-only-access]]
 === [xpack]#Read only access#
```

Comment:
Totally outside the parameters of this PR, but could we make this an attribute? I've been sticking this text in my PRs too, so I'm wondering if we could keep and maintain it in one place?